enginetest/memory_engine_test.go

```diff
@@ -524,6 +524,7 @@ func TestUserPrivileges(t *testing.T) {
 }
 
 func TestUserAuthentication(t *testing.T) {
+	queries.ServerAuthTests = queries.ServerAuthTests[:2]
```

Comment:
?

Reply:
At least add a comment

Better yet, make a different var here with a descriptive name

Reply:
Leaving this in there was an accident lol, this was just to speed up testing by not running all of the tests at once